app/views/jobseekers/registrations/new.html.slim

```diff
@@ -1,9 +1,11 @@
 - content_for :page_title_prefix, t(".title")
 
+= govuk_back_link text: t("buttons.back"), href: new_jobseeker_session_path
+
 .govuk-main-wrapper
   .govuk-grid-row
     .govuk-grid-column-two-thirds
-      h1.govuk-heading-xl = t(".title")
+      h1.govuk-heading-l = t(".title")
```

Comment:
Deliberate? I wonder if the title should remain `xl` so hiring staff are more likely to notice they're on the wrong page.

Reply:
This is what Jesse designed:

https://teaching-vacancies-prototype.london.cloudapps.digital/prototypes/application/journey/sign-in

You can test it pixel by pixel :)